go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/mimir
 
-go 1.22
+go 1.22.0
```

Comment:
what effect does this have?

Reply:
From reading https://go.dev/doc/toolchain, I don't think there's any difference between the two.

Reply:
I'm not sure, `go mod tidy` sets it